ansible/roles/debops.discourse/defaults/main.yml

```diff
@@ -0,0 +1,686 @@
+                          ansible_local.root.src|d())
+                      else "/usr/local/src") + "/discourse" }}'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# System user, group, additional groups [[[
+# -----------------------------------------
+
+# .. envvar:: discourse_user [[[
+#
+# System UNIX account used by the Discourse application. It will be visible in the
+# :command:`git+ssh` remote URLs.
+discourse_user: '{{discourse__instance_name}}'
+
+                                                                   # ]]]
+# .. envvar:: discourse_group [[[
+#
+# System UNIX group used by the Discourse application.
+discourse_group: '{{discourse__instance_name}}'
+
+                                                                   # ]]]
+# .. envvar:: discourse_user_append_groups [[[
+#
+# List of additional system groups to add to the Discourse user account.
+# The ``sshusers`` UNIX group is used in DebOps to limit SSH access. See the
+# :ref:`debops.system_groups` role for more details.
+discourse_user_append_groups: []
+
+                                                                   # ]]]
+# .. envvar:: discourse__shell [[[
+#
+# The default shell used by the Discourse UNIX account.
+discourse__shell: '/bin/bash'
+                                                                   # ]]]
+                                                                   # ]]]
+# Internal application options [[[
+# --------------------------------
+
+# .. envvar:: discourse_unicorn_workers [[[
+#
+# Number of Unicorn workers to start.
+# If you only have 1GB RAM, use 2 workers. Otherwise increase to 3 or 4.
+discourse_unicorn_workers: 2
+
+                                                                   # ]]]
+# .. envvar:: discourse_unicorn_connection [[[
+#
+# Connection type for unicorn (choices: socket, port)
+discourse_unicorn_connection: 'socket'
+                                                                   # ]]]
+# .. envvar:: discourse_unicorn_port [[[
+#
+# Unicorn port on ``localhost`` interface. Unused if
+# :var:`discourse_unicorn_connection` is set to ``socket``.
+discourse_unicorn_port: '18082'
+
+                                                                   # ]]]
+                                                                   # ]]]
+# Compatibility workarounds [[[
+# -----------------------------
+
+# .. envvar:: discourse_support_filesystem_acl [[[
+#
+# Enable or disable ACL configuration for the webserver
+discourse_support_filesystem_acl: True
+                                                                   # ]]]
+                                                                   # ]]]
+# Discourse source code [[[
+# -------------------------
+
+# .. envvar:: discourse_git_repo [[[
+#
+# URL of the Discourse repository.
+discourse_git_repo: 'https://github.com/discourse/discourse.git'
```

Comment:
Discourse releases are... unsigned. *sigh* And the repository does not have the "Issues" page. But I guess that after so many releases upstream is completely uninterested it making GPG-signed releases, since they focus on Docker as a distribution method.

Does Discourse upstream use stable branches in the repository for updates? If not, perhaps you should consider deploying a specific `git` tag instead, upgrades could be done after reviewing that nothing breaks and new upstream releases can be tracked using the `watch` files.